tests/test_notebook.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+"""
+Author : Dogeek
```

Comment:
Please change the file docstring to either a single line or the standard format used in the other files in the repository.